devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -103,6 +103,10 @@ def installed() {
 	// Device-Watch pings if no device events received for 1 hour (checkInterval)
 	sendEvent(name: "checkInterval", value: 1 * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
 
+	if (isSamsungLock()) { // Samsung locks won't pair when locked, so it must be unlocked
```

Comment:
Won't pair, what does that mean in respect to Z-Wave? Secure Inclusion?

Reply:
@kianooshST Sorry, the comment was a little vague. I updated it in the code. Basically, you can't enter the pairing menu if the lock is locked. The idea being how can you be inside and outside at the same time. :P